machine_management/control_plane_machine_management/cpms-changing-openstack-flavor-type.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+// * machine_management/control_plane_machine_management/cpmso-using.adoc
+
+ifeval::["{context}" == "recommended-control-plane-practices"]
+:scale-host:
+endif::[]
+ifeval::["{context}" == "cpmso-using"]
+:cpmso-using:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="cpms-changing-openstack-flavor-type_{context}"]
+= Changing the OpenStack Nova Flavor by using a control plane machine set
+
+You can change the AOpenStack Nova Flavor that your control plane machines use by updating the specification in the control plane machine set custom resource (CR).
```

Comment:
{rh-openstack-first} for first use of RHOSP. Otherwise, {rh-openstack}. 

If you're talking about the platform in a broad sense, which happens sometimes in these docs, yeah, just use "OpenStack," but that's not the case here.

Reply:
Ack.